rfc/rfc019-employee-identity-means.md

```diff
@@ -0,0 +1,187 @@
+# RFC019 Employee Identity Authentication Means
+
+|                           |                 |
+|:--------------------------|:----------------|
+| Nuts foundation           | W.M. Slakhorst  |
+| Request for Comments: 019 | Nedap           |
+|                           | S. van der Vegt |
+|                           | Nedap           |
+|                           | April 2023      |
+
+## Employee Identity Authentication Means
+
+### Abstract
+
+The NutsEmployeeIdentity method is an authentication _means_ which allows an organisation to assert the identity of the current user.
+The organisation uses the information of the current user session to present the user with a statement about its identity and asks the user to confirm this.
+After confirmation this bundle of information is signed with the organisation's private key.
+
+This method makes it possible to assert a user identity without the need for a personal authentication means.
+This puts the organisation in the role of identity issuer instead of a trusted third party.
+A use-case can choose to allow this authentication under certain (low risk) circumstances which still require a person to be authenticated in order to comply with the applicable legislation.
+
+This RFC is an addition to the means listed in [RFC002](./rfc002-authentication-token.md#7-supported-means)
+
+### Status of document
+
+This document is currently in draft.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1.  Introduction
+
+Introducing new authentication means in an organizational setting can have quite an impact, even more when mainstream identity providers have not yet adjusted to new standards.
```

Comment:
> even more when mainstream identity providers have not yet adjusted to new standards.

How applicable is this? IRMA vs mainstream identity providers? And IRMA isn't a standard?

Either make it more specific (IRMA) or abstract: it's about an authentication means which is under sole control of the user (self-sovereign).

Maybe just use "self-sovereign identity"?